Extension/CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 # C/C++ for Visual Studio Code Change Log
 
-## Version 0.19.1-insiders2: October 12, 2018
+## Version 0.19.1-insiders2: October 16, 2018
 * Add IntegratedTerminal support for Linux and Windows. [#35](https://github.com/microsoft/vscode-cpptools/issues/35)
 * Unify Visual Studio Code debug protocol parsing by using a shared library with Visual Studio.
+* Fix `#include` completion not working with `compile_commands.json` or custom configuration providers. [#2242](https://github.com/Microsoft/vscode-cpptools/issues/2242)
```

Comment:
`compile_commands.json` **and** custom configuration providers

Reply:
I think?

Reply:
Sure, "and" seems better to me.